apps/aecore/src/aec_chain.erl

```diff
@@ -152,26 +160,26 @@ common_ancestor(Hash1, Hash2) ->
                     {common_ancestor, Hash1, Hash2}).
 
 get_transactions_between(Hash1, Root) ->
-    try get_transactions_between(Hash1, Root, []) of
+    {ok, GenesisHeader} = genesis_header(),
+    GenesisHash = aec_headers:hash_header(GenesisHeader),
```

Comment:
aec_headers:hash_header returns {ok, Hash}, right? If you fix this in the code you will solve https://www.pivotaltracker.com/n/projects/2124891/stories/152954773 as well.